app/models/Search.java

```diff
@@ -263,14 +263,18 @@ public boolean doingScrollScanNow() {
 		return doingScrollScanNow;
 	}
 
-	private void bulk(final Request request, final Serialization serialization) {
+	private void bulk(final Request request, final Serialization serialization,
+			final String scroll) {
 		boolean JSON_LD = serialization.equals(Serialization.JSON_LD);
+		boolean RDF_XML = serialization.equals(Serialization.RDF_XML);
 		try {
 			long lastTime = Calendar.getInstance().getTimeInMillis();
-			SearchResponse searchResponse = startInitialResponse();
+			SearchResponse searchResponse = startInitialResponse(scroll);
 			final SearchHits hits = getTotalHits(searchResponse);
 			if (JSON_LD)
 				getMessageOut().write("[");
+			if (RDF_XML)
+				getMessageOut().write("<root>");
```

Comment:
You should validate the results with e.g. 'xmllint' for xml and 'rapper' for rdf.
The validation shows that it's ok to wrap the '<rdf:RDF>' entries in whatever element while it's not ok to lack a root element (as it is in our current production - I don't know where you see a root element there?)